native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -1,20 +1,72 @@
 import UIKit
 import SharedCode
 
-class ViewController: UIViewController {
-
-    @IBOutlet private var label: UILabel!
-
+class ViewController: UIViewController, UIPickerViewDataSource, UIPickerViewDelegate {
+    @IBOutlet weak var originSpinner: UIPickerView!
+    @IBOutlet weak var destinationSpinner: UIPickerView!
+    @IBOutlet weak var submitButton: UIButton!
+    
+    private var stations: [String] = [String]()
+    
     private let presenter: ApplicationContractPresenter = ApplicationPresenter()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         presenter.onViewTaken(view: self)
+        
+        originSpinner.delegate = self
+        originSpinner.dataSource = self
+        
+        destinationSpinner.delegate = self
+        destinationSpinner.dataSource = self
     }
+    
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        return stations.capacity
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        return stations[row]
+    }
+    
+
 }
 
```

Comment:
Could we add an extension here to handle picker, e.g. 
`extension ViewController: UIPickerViewDataSource, UIPickerViewDelegate { .. }` and move the functions to do with the picker inside here. Would also be nice to have a function like setUpPicker in here which is called from viewDidLoad() and contains the lines:
```
originSpinner.delegate = self
         originSpinner.dataSource = self

         destinationSpinner.delegate = self
         destinationSpinner.dataSource = self
```
to avoid viewDidLoad getting bloated with loads of stuff